backend/src/test/kotlin/fr/gouv/cacem/monitorenv/infrastructure/database/repositories/JpaReportingITests.kt

```diff
@@ -2,15 +2,21 @@ package fr.gouv.cacem.monitorenv.infrastructure.database.repositories
 
 import fr.gouv.cacem.monitorenv.domain.entities.VehicleTypeEnum
 import fr.gouv.cacem.monitorenv.domain.entities.reporting.ReportingEntity
+import fr.gouv.cacem.monitorenv.domain.entities.reporting.ReportingTypeEnum
 import fr.gouv.cacem.monitorenv.domain.entities.reporting.SourceTypeEnum
 import fr.gouv.cacem.monitorenv.domain.entities.reporting.TargetTypeEnum
-import java.time.ZonedDateTime
+import fr.gouv.cacem.monitorenv.domain.exceptions.NotFoundException
 import org.assertj.core.api.Assertions.assertThat
 import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+import org.locationtech.jts.geom.MultiPolygon
+import org.locationtech.jts.io.WKTReader
 import org.springframework.beans.factory.annotation.Autowired
 import org.springframework.boot.test.context.SpringBootTest
 import org.springframework.data.domain.Pageable
 import org.springframework.transaction.annotation.Transactional
+import java.time.ZonedDateTime
+import java.util.UUID
 
 @SpringBootTest(properties = ["monitorenv.scheduling.enable=false"])
```

Comment:
Vous n'avez pas de Scheduling sur Env

```suggestion
@SpringBootTest
```

Reply:
si, on a un scheduling pour l'archivage des signalements. Par contre il y a une typo, un `enabled` serait mieux non?

Reply:
Ah okey je ne savais pas!